style.css

```diff
@@ -101,4 +101,27 @@ button {
     text-transform: uppercase;
     background: var(--accent-color);
     color: #fff;
+}
+
+
+/*
+**Desktop draft
```

Comment:
Not sure we need this comment here :o

Reply:
Ahah, no we don't need this, was just for information but can be deleted ^^